orca-core/src/main/java/com/netflix/spinnaker/orca/pipeline/expressions/functions/StageExpressionFunctionProvider.java

```diff
@@ -186,6 +199,7 @@ public static String judgment(PipelineExecution execution, String id) {
   }
 
   /** Alias to judgment */
+  @SuppressWarnings("unused")
   public static String judgement(PipelineExecution execution, String id) {
```

Comment:
Is this used dynamically by the SpEL expressions? I see 8 instances of the extra `e` spelling and 100+ without mostly 
> Judgment can also be spelled "judgement," and usage experts have long disagreed over which spelling is the preferred one. Henry Fowler asserted, "The OED [Oxford English Dictionary] prefers the older and more reasonable spelling. 'Judgement' is therefore here recommended." William Safire held an opposite opinion, writing, "My judgment is that Fowler is not to be followed." "Judgement" is in fact the older spelling, but it dropped from favor and for centuries "judgment" was the only spelling to appear in dictionaries. That changed when the OED (Fowler's source) was published showing "judgement" as an equal variant. Today, "judgment" is more popular in the U.S., whereas both spellings make a good showing in Britain.

Reply:
This has been in here for a really long time and would be potentially embedded in pipeline configurations.. I just stuck the suppress unused to get rid of an IDE warning (these never show as used because they are used via SpEL).

I think we stick with this indefinitely since it is out in the wild